package.json

```diff
@@ -84,12 +84,14 @@
     "**/@types/angular": "^1.6.56",
     "**/@types/hoist-non-react-statics": "^3.3.1",
     "**/@types/chai": "^4.2.11",
-    "**/cypress/@types/lodash": "^4.14.155",
+    "**/cypress/@types/lodash": "^4.14.159",
+    "**/cypress/lodash": "^4.17.20",
     "**/typescript": "3.9.5",
     "**/graphql-toolkit/lodash": "^4.17.15",
     "**/hoist-non-react-statics": "^3.3.2",
     "**/isomorphic-git/**/base64-js": "^1.2.1",
     "**/image-diff/gm/debug": "^2.6.9",
+    "**/load-grunt-config/lodash": "^4.17.20",
```

Comment:
load-grunt-config includes lodash as a transitive dependency, locked at "4.17.15". Here I resolve it to "^4.17.20".